python/semantic_kernel/connectors/ai/prompt_execution_settings.py

```diff
@@ -5,20 +5,23 @@
 from semantic_kernel.kernel_pydantic import KernelBaseModel
 
 
-class AIRequestSettings(KernelBaseModel):
-    """Base class for AI request settings.
+class PromptExecutionSettings(KernelBaseModel):
+    """Base class for prompt execution settings.
 
-    Can be used by itself or as a base class for other request settings. The methods are used to create specific request settings objects based on the keys in the extension_data field, this way you can create a generic AIRequestSettings object in your application, which get's mapped into the keys of the request settings that each services returns by using the service.get_request_settings() method.
+    Can be used by itself or as a base class for other prompt execution settings. The methods are used to create
```

Comment:
Extra white space added?

Reply:
I manually broke the line into multiple lines to fit in 120 characters. I can revert it if we think docstrings shouldn't follow that rule.

Reply:
Oh, it was a single line before, that's why there's so much white space. No worries, I like cutting on 120 chars max.